pina/solvers/pinn.py

```diff
@@ -135,8 +148,8 @@ def training_step(self, batch, batch_idx):
         total_loss = sum(condition_losses)
 
         self.log('mean_loss', float(total_loss / len(condition_losses)), prog_bar=True, logger=True)
-        for condition_loss, loss in zip(condition_names, condition_losses):
-            self.log(condition_loss + '_loss', float(loss), prog_bar=True, logger=True)
+        # for condition_loss, loss in zip(condition_names, condition_losses):
```

Comment:
why did we remove these loggers?

Reply:
Ok, we can log at each full dataloader iteration by taking the mean fo the batches losses. Let's fix it later